### PR TITLE
Set up Bazel build for the backend

### PR DESCRIPTION
**Pull request by mortenmj**

### CODE REVIEW

**Comment by mortenmj** [comment]
@trey-ivy I tried rebasing this on #3 and had a couple of test failures. Would you mind if we merged that PR first, so to get some basic CI up and running?

File: .github/workflows/master.yaml
Comment (trey-ivy) [inline .github/workflows/master.yaml:1]: ❤️

**Review by trey-ivy — APPROVED** [review]
This looks great @mortenmj .  I saw your comment about tests in slack too and yes, i will need to address in [#2 ](https://github.com/buildbarn/bb-portal/pull/2) but this is awesome.  👍🏻 Are you able to update and add some documentation about using this tooling in a subsequent PR?

File: .github/workflows/master.yaml
Comment (mortenmj) [reply .github/workflows/master.yaml:1]: > This looks great @mortenmj . I saw your comment about tests in slack too and yes, i will need to address in https://github.com/buildbarn/bb-portal/pull/2but this is awesome. 👍🏻 Are you able to update and add some documentation about using this tooling in a subsequent PR?

Yes; I'll get on that next week

**Pull request merged**